<launch>

  <node pkg="laser_scan_matcher" type="laser_scan_matcher_node" 
    name="laser_scan_matcher_node" output="screen">
    <remap from="scan" to="/scan_filtered_left"/>
    <remap from="imu/data" to="mpu6050"/>
    <param name="fixed_frame" value="odom"/>
    <param name="base_frame" value="base_footprint"/>
    <param name="use_imu" value="false"/>
  </node>

</launch>
